vars/infra.groovy

```diff
@@ -40,7 +40,7 @@ Object checkoutSCM(String repo = null) {
     }
 
     if (env.BRANCH_NAME) {
-        checkout scm
+        checkout repo
```

Comment:
`checkout scm` is the way of doing this, have you got a link for `checkout repo` I've never heard of it